include/engine/guidance/assemble_leg.hpp

```diff
@@ -182,6 +182,13 @@ inline RouteLeg assembleLeg(const datafacade::BaseDataFacade &facade,
         duration = std::max(0, duration);
     }
 
+    // Add start and stop penalties
+    if (distance > 0)
```

Comment:
this is too un-nuanced. we should add a constant distance at which penalties make sense. maybe something like 50m?